nilearn/decoding/space_net.py

```diff
@@ -806,9 +807,22 @@ def fit(self, X, y):
         mask_volume = _get_mask_volume(self.mask_img_)
         if mask_volume > MNI152_BRAIN_VOLUME:
             warnings.warn(
-                "Brain mask is bigger than volume of standard brain!")
-        self.screening_percentile_ = self.screening_percentile * (
-            mask_volume / MNI152_BRAIN_VOLUME)
+                "Brain mask is bigger than the volume of a standard "
+                "humain brain. Strange things might happen.",
+                stacklevel=2)
+        elif mask_volume < .005 * MNI152_BRAIN_VOLUME:
```

Comment:
Or maybe if this test passes, then check the `screening_percentile` and force it to `100` if less (with the appropriate warning) ?